src/functions/template-tags/link.php

```diff
@@ -136,11 +136,16 @@ function tribe_get_events_link( $context = 'href' ) {
 /**
  * Gets a view permalink.
  *
- * @since 5.7.0
+ * Generates the permalink for a specific view based on the provided slug and optional term.
  *
- * @param bool|int|null $term
+ * @since 5.7.0
+ * 
+ * @hook tribe_get_view_permalink        Filters the overall view permalink.
+ * @hook tribe_get_{slug}_view_permalink Filters the specific view permalink.
  *
- * @return string $permalink
+ * @param string        $slug      The slug of the view for which the permalink is being generated.
+ * @param bool|int|null $term      Optional. The term associated with the view. Default is null.
+ * @param string        $permalink The generated permalink for the view.
```

Comment:
I think G meant the return is missing - it got changed to a param by accident:
```suggestion
 * 
 * @return string        $permalink The generated permalink for the view.
```

Reply:
Check the spacing on that line I suggested - it might be off 😉

Reply:
Aha! That's right. Thanks